_posts/2023-10-22-specify-file-audit-log-encoding.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: 'Change the default encoding of the audit log file in WildFly 29'
+date: 2023-10-22
```

Comment:
@ivassile Thank you LGTM! This just needs to have a date updated and the confligc resolved and we can merge